src/app/_components/BorderHighlight/index.module.scss

```diff
@@ -0,0 +1,49 @@
+@use '@scss/common.scss' as *;
+
+.container {
```

Comment:
Can container and backgroundContainer be combined here? And then you could remove the backgroundContainer div as well.